README.md

```diff
@@ -119,17 +125,47 @@ NOTE: These steps will cover more aspects over time. For now it focuses on testi
     - Reload your server for the new env vars to apply and resend the verification payloads
     - You will have to do this with multiple settings, thus, you will have to repeat reloading your server as you add new variables
 
-1.  Create a new GitHub repository at `<YOUR_GITHUB_USERNAME>/eng-pipes-dev`.
-1.  Create a [new GitHub App](https://github.com/settings/apps/new)
+1.  Create a new personal GitHub organization at `<YOUR_GITHUB_USERNAME>/eng-pipes-org`
+
+    - In that organization, create a [new GitHub App](https://github.com/settings/apps/new)
     - Set the webhook to your ngrok tunnel with the GH route (e.g. `<NGROK_INSTANCE>/webhooks/github`)
     - Place the secrets in your `.env` (see [Setup Secrets](#setup-secrets) below)
     - Go to the `Permissions & events` sidebar menu entry of the GitHub app configuration, and grant maximum non-`Admin` access (`Read and write` where possible, `Read only` everywhere else) for every line in `Repository permissions` (NOTE: We use a more constrained permission-set in production, but for initial setup enabling maximal permissions is fine; permissions can be whittled down later as needed.)
     - For `Organization permissions`, grant `Read and write` for `Members` and `Projects`
     - In the `Subscribe to events` section, check every possible box
-1.  Go to `Install App` in the sidebar menu of the GitHub app configuration, and install the app for your GitHub account.
-    - When prompted, choose `Only select repositories`, and pick only the `eng-pipes-dev` repository you created in step 4
+    - Go to `Install App` in the sidebar menu of the GitHub app configuration, and install the app for your personal GitHub organization
+    - When prompted, choose `All repositories`
+
+1.  In your personal organization, create a new personal GitHub repository at `eng-pipes-org/eng-pipes-dev`.
+
+    - After the organization has been created, go to its `Settings`, then select `Personal access tokens` from the sidebar to enable tokens
+    - In the setup menu, select `Allow access via fine-grained personal access tokens`, then `Do not require administrator approval`, and finally `Allow access via personal access tokens (classic)`
+    - Now, go to your own account `Settings`, select `Developer Settings` from the sidebar, then choose to `Generate personal access token (classic)`
+    - Title the new token `Eng-pipes development token`, give this token 90 days until expiration, and enable the following permissions: `admin:org`, `admin:org_hook`, `admin: repo_hook`, `project`, `public_repo`, `repo:status`, and `repo_deployment`.
```

Comment:
Do we really need all of these perms? The one I'm using only has `read:org` and `read:user`, it's a classic token with no expiration date, and in my `.env` I have this note: "For org membership."

Reply:
Ok, changed to those instead. I just set those permissions because those sounded like things that may plausibly be useful for it to do in the future, but we can always change the instructions then.

Done.